overdrive.py

```diff
@@ -445,24 +448,20 @@ def _get_book_list_page(self, link, rel_to_follow='next'):
            one book.
         `next_link` is a link to the next page of results.
         """
+        extractor_class = extractor_class or OverdriveRepresentationExtractor
         # We don't cache this because it changes constantly.
         status_code, headers, content = self.get(link, {})
-        if isinstance(content, str):
+        if isinstance(content, (bytes, str)):
```

Comment:
Here's the other bug fix. It's now the same as code we have several times elsewhere in this file.

This is some of the first code I wrote for the project, back in 2014, and these methods for extracting information from Overdrive API responses aren't tested very well, nor designed to be testable.